frontend/src/classes/GovsgCampaign.ts

```diff
@@ -36,6 +40,7 @@ export class GovsgCampaign extends Campaign {
     this.numRecipients = input['num_recipients']
     this.forSingleRecipient = input['govsg_templates']?.for_single_recipient
     this.paramMetadata = input['govsg_templates']?.param_metadata
+    this.languages = input['govsg_templates']?.languages ?? []
```

Comment:
Am I correct to say that don't have to provide a default language because we already set a default value when the API is called?

Reply:
Which API?

But yes, there is a default value for `language_code` in `GovsgMessage`. As a result, I also specified a default value in the frontend (there was this line that says `fieldsToRender ...`).

Reply:
Im referring to the `/send-single` route. I saw that a default value is added in the input checks backend/src/govsg/routes/govsg-campaign.routes.ts

Reply:
I see. Yes, you are correct.

And we also have this in the frontend:

```javascript
  const [data, setData] = useState<Record<string, string>>(
    fieldsToRender.reduce((cul, f) => ({ [f.id]: '', ...cul }), {
      recipient: '',
      languageCode: WhatsAppLanguages.english,
    })
  )
```